tests/foreman/smoke/test_api_smoke.py

```diff
@@ -784,13 +782,8 @@ def test_ping(self):
         @Assert: Overall and individual services status should be 'ok'.
 
         """
-        response = client.get(
-            entities.Ping().path(),
-            auth=get_server_credentials(),
-            verify=False,
-        )
-        response.raise_for_status()
-        self.assertEqual(response.json()['status'], u'ok')  # overall status
+        response = entities.Ping().search_json()
+        self.assertEqual(response['status'], u'ok')  # overall status
```

Comment:
This is one of the few places where we have a real need to drop down and use a lower-level `search_*` method.